components/searchBar/index.js

```diff
@@ -104,26 +116,52 @@ export default class searchBar extends Component {
       <View style={styles.container}>
         <View>
           <SearchBar
-            searchIcon={<Icon navigation={this.props.navigation} />}
+            platform="android"
+            autoCorrect={false}
+            padding={5}
+            returnKeyType="search"
             lightTheme
+            containerStyle={{
+              borderRadius: 10,
+              borderWidth: 1,
+              height: 45,
+
```

Comment:
There are a lot of useless spaces in the PR, could you run the linter and make sure that they are not there?

Reply:
Linter did not detect that white space, will manually look for them and fix